playground/mocks/config/responseConfig.js

```diff
@@ -18,7 +18,7 @@ const idx = {
   ],
 
   '/idp/idx/introspect': [
-    'identify',
+    // 'identify',
```

Comment:
remember to revert this change before you open PR for review

Reply:
Will do. Enabled just the relevant responses for easy bug repro